code/modules/jobs/job_types/paramedic.dm

```diff
@@ -26,9 +26,9 @@
 	ears = /obj/item/radio/headset/headset_med
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
 	uniform = /obj/item/clothing/under/rank/medical/paramedic
-	mask = /obj/item/clothing/mask/cigarette
 	shoes = /obj/item/clothing/shoes/jackboots
 	head = /obj/item/clothing/head/soft/emt
+	mask = null //Skyrat Edit
```

Comment:
oh god dont use null
Just do //mask = /obj/item/clothing/mask/cigarette - Skyrat Edit
This messes with loadout spawning

Reply:
Useroth himself told me to do it like that lmao